src/tools/pkcs15-init.c

```diff
@@ -2627,8 +2630,10 @@ handle_option(const struct option *opt)
 	case OPT_NO_SOPIN:
 		opt_no_sopin = 1;
 		break;
-	case OPT_NO_PROMPT:
-		opt_no_prompt = 1;
+	case OPT_USE_PINPAD_DEPRECATED:
+		fprintf(stderr, "--no-prompt have been deprecated and will be removed. Use instead '--use-pinpad'.\n");
```

Comment:
Please change the messages to "'--no-prompt' is deprecated , use '--use-pinpad' instead.\n